contrib/utilities/update-copyright

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2015 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+
+# Purpose: Update the copyright year of every file based on the last
+#          modification recorded in the git logs
```

Comment:
I think the problem is, if the last time a file was changed was in 2014 your script will update the file to something like 2008-20014. Now if you do a commit and rerun the script, you will get 2008-2015 because the copyright was updated in 2015.